modules/saas/apps/saml-saas/src/main/java/com/liferay/saml/saas/internal/configuration/SaasConfiguration.java

```diff
@@ -24,18 +24,19 @@
 @ExtendedObjectClassDefinition(
 	generateUI = false, scope = ExtendedObjectClassDefinition.Scope.COMPANY
 )
-@Meta.OCD(id = "com.liferay.saas.configuration.SaasConfiguration")
+@Meta.OCD(id = "com.liferay.saml.saas.configuration.SAMLSaasConfiguration")
```

Comment:
Should we rename this class to _SAMLSaasConfiguration_?

Reply:
Probably `SamlSaasConfiguration`

Reply:
hey... yes... thanks for catching that